File: Assignment_1/new_ele.py
n=int(input("Enter size of array:"))
arr=[]
for i in range(n):
    arr.append(int(input("Enter element:")))
ele=int(input("Enter new element:"))
pos=int(input("Enter position of element:"))
arr.insert(pos, ele)
print("Modified array:",end=' ')
for i in range(ln):
    print(arr[i],end=' ')
print()

File: Assignment_1/new_ele.c
#include <stdio.h>
int main()
{
    int n;
    printf("Enter size of array:");
    scanf("%d",&n);
    int arr[n+1];
    for(int i=0;i<n;i++)
    {
        printf("Enter element:");
        scanf("%d",&arr[i]);
    }
    int ele,pos;
    printf("Enter element and position:");
    scanf("%d %d",&ele,&pos);
    arr[pos]=ele;
    printf("Modified array:");
    for(int i=0;i<n;i++)
    {
        printf("%d ",arr[i]);
    }
    printf("\n");
    
    return 0;
}